sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -609,6 +494,17 @@ def test_pred_int_tag(self, estimator_instance):
         if isinstance(f, _DelegatedForecaster):
             return None
 
+        # we skip the PytorchForecastingNBeats,
```

Comment:
this should not be done in the suite tests - instead, use `tests._config` to add the exception

Reply:
I tried to add `"PytorchForecastingNBeats": ["test_pred_int_tag"]` to `sktime.tests._config.py::EXCLUDED_TESTS`, however, `check_estimator(PytorchForecastingNBeats, raise_exceptions=True)` didn't work.

Reply:
yes, the `check_estimator` ignores the `_config` skips, that is intended design.

Reply:
It means that `check_estimator(PytorchForecastingTFT)` will fail then. Is it intended design?